PL-Policy.md

```diff
@@ -20,4 +20,55 @@ See the **_Applicability_** section of the [GSA IT Security Policy](http://www.g
+
+The determining factors of Confidentiality, Integrity, and Availability of the cloud.gov system are its FIPS 199 information types, which are listed in Section 2 of its SSP. The resulting Security Categorization, FIPS 199 Moderate Impact, governs the cloud.gov security architecture, which NIST SP 800-27A defines as “A description of security principles and an overall approach for architecture complying with the principles that drive the system design.” The information security architectural approach is documented in the cloud.gov System Security Plan, which describes implementation of said principles. 
+
+The cloud.gov platform leverages IaaS via Amazon Web Services, which received a FedRAMP ATO in June 21, 2016.  This allows the cloud.gov platform to inherit some security controls such as physical security and share responsibility of other controls such as media protection. For web applications the 18F Cloud Operations team ensures that a web vulnerability scanner (OWASP Zap) is run on a monthly basis. 18F web applications use industry best practices and federal hardening guidelines for web servers and application services like Java.  
+
+The SSP references an attached Services Table in describing control AC-20, “Use of External Information Systems”. The attachment comprehensively lists external services incorporated into cloud.gov operations, as well as information about the kinds of information exchanged with them and assumptions about the security and sensitivity of that information.
+The cloud.gov ISSO, cloud.gov System Owner, and FedRAMP JAB review applicable security architectures prior to major implementations or security assessments. The cloud.gov ISSO(s) reviews and updates the information security architecture within the System Security Plan on an annual basis or when a significant change takes place to reflect updates in the enterprise architecture. Due to the dynamic and elastic nature of cloud computing, the operations team monitors real-time updates of its information security architecture using the AWS Management Console and other management tools. 
+The cloud.gov operations team follows the risk management framework (RMF) which includes conducting annual risk assessments for its information systems and infrastructure. Any changes are then updated in systems security plans, plan of actions and milestones POAMs, security assessment reports (SAR).
+
+All changes to the cloud.gov platform are routed through the 18F Change Control process using the GitHub ticketing and tracking system which monitors changes including, but not limited to, ensuring that information security architecture changes are appropriately reflected in updates to system security plans (SSP) Standards operating procedures (SOP) and other security architecture related documentation.  
```

Comment:
"system security plans (SSP) Standards operating procedures (SOP)" needs editing (probably in the SSP as well)